Jenkinsfile

```diff
@@ -296,103 +320,130 @@ pipeline {
+                        ShutdownCluster('cluster2')
+                    }
+                }
+                stage('cluster3') {
+                    steps {
+                        CreateCluster('cluster3')
+                        runTest('init-deploy', 'cluster3', '8.0', 40)
+                        runTest('limits', 'cluster3', '8.0', 30)
+                        runTest('monitoring-2-0', 'cluster3', '8.0', 30)
+                        ShutdownCluster('cluster3')
+                    }
+                }
+                stage('cluster4') {
+                    steps {
+                        CreateCluster('cluster4')
+                        runTest('proxysql-sidecar-res-limits', 'cluster4', '8.0', 30)
+                        runTest('tls-issue-self','cluster4', '8.0', 25)
+                        runTest('tls-issue-cert-manager','cluster4', '8.0', 25)
+                        runTest('tls-issue-cert-manager-ref','cluster4', '8.0', 25)
+                        runTest('validation-hook','cluster4', '8.0', 10)
+                        ShutdownCluster('cluster4')
+                    }
+                }
+                stage('cluster5') {
+                    steps {
+                        CreateCluster('cluster5')
+                        runTest('scaling', 'cluster5', '8.0', 30)
+                        runTest('scaling-proxysql', 'cluster5', '8.0', 30)
+                        runTest('security-context', 'cluster5', '8.0', 50)
+                        ShutdownCluster('cluster5')
+                    }
+                }
+                stage('cluster6') {
+                    steps {
+                        CreateCluster('cluster6')
+                        runTest('storage', 'cluster6', '8.0', 30)
+                        runTest('upgrade-consistency', 'cluster6', '8.0', 50)
+                        runTest('proxy-protocol','cluster6', '8.0', 30)
+                        ShutdownCluster('cluster6')
+                    }
+                }
+                stage('cluster7') {
                     steps {
-                        CreateCluster('upgrade')
-                        runTest('upgrade-haproxy', 'upgrade', '8.0')
-                        ShutdownCluster('upgrade')
-                        CreateCluster('upgrade')
-                        runTest('upgrade-proxysql', 'upgrade', '8.0')
-                        ShutdownCluster('upgrade')
-                        CreateCluster('upgrade')
-                        runTest('smart-update', 'upgrade', '8.0')
-                        runTest('upgrade-consistency', 'upgrade', '8.0')
-                        ShutdownCluster('upgrade')
+                        CreateCluster('cluster7')
+                        runTest('restore-to-encrypted-cluster', 'cluster7', '8.0', 50)
+                        runTest('pitr', 'cluster7', '8.0', 75)
+                        runTest('affinity', 'cluster7', '8.0', 20)
+                        ShutdownCluster('cluster7')
```

Comment:
Did we set this time with a margin ? Just curious (=

Reply:
Yes, there is a margin, but it's not fixed - it depends on the test.